src/psyclone/tests/test_files/dynamo0p3/testkern_write_w2v_w1_mod.f90

```diff
@@ -30,33 +30,32 @@
 ! ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 ! POSSIBILITY OF SUCH DAMAGE.
 ! -----------------------------------------------------------------------------
-! Authors R. W. Ford and A. R. Porter, STFC Daresbury Lab
+! Author R. Ford STFC Daresbury Lab
```

Comment:
This previously had A. R. Porter as an author as well as me. Could you please put him back.

Reply:
Oops, sorry about that! He is back.